src/smtsolvers/CMakeLists.txt

```diff
@@ -41,13 +34,12 @@ if(PRINT_UNITS)
 endif()
 
 target_sources(proof
-    PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Proof.h"
-    PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/Proof.cc"
-)
-install(FILES TheoryInterpolator.h
-        DESTINATION ${INSTALL_HEADERS_DIR})
+	PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/Proof.h"
+	PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/Proof.cc")
 
+install(FILES TheoryInterpolator.h
+	 DESTINATION ${INSTALL_HEADERS_DIR})
 
-install(FILES SimpSMTSolver.h CoreSMTSolver.h ScatterSplitter.h
-DESTINATION ${INSTALL_HEADERS_DIR})
 
+install(FILES SimpSMTSolver.h CoreSMTSolver.h
+		DESTINATION ${INSTALL_HEADERS_DIR})
```

Comment:
The changes in the indentation seem strange.

Reply:
Yeah, I have no idea what happened!
I tried to fix it but, every time would get more strange